wagtail/permission_policies/collections.py

```diff
@@ -20,60 +22,67 @@ def _get_permission_objects_for_actions(self, actions):
             content_type=self._content_type, codename__in=permission_codenames
         )
 
+    def _get_user_permission_objects_for_actions(self, user, actions):
+        """
+        Get a set of the user's GroupCollectionPermission objects for the given actions
+        """
+        permission_codenames = {
+            get_permission_codename(action, self.auth_model._meta) for action in actions
+        }
+        return {
+            group_permission
+            for group_permission in self.get_cached_permissions_for_user(user)
+            if group_permission.permission.codename in permission_codenames
+        }
+
+    def get_all_permissions_for_user(self, user):
```

Comment:
This method is never used directly, but it's used via `get_cached_permissions_for_user()` that was added to `BasePermissionPolicy` in 193b53af190ad8e509c6773f4d26f724018de5fd.

The `get_cached_permissions_for_user()` method is used in `_get_user_permission_objects_for_actions()` that filters the cached objects by the actions.